tests/lib.rs

```diff
@@ -200,21 +214,55 @@ fn test_setup_seed_config() {
         .stdout(contains("Successfully created seed config file"));
 }
 
+/// This test is to show that there is a default path for custom_pages_dir if it is not defined in
+/// the config.toml
+#[test]
+fn test_show_paths_custom_pages_not_in_config() {
+    use app_dirs::{get_app_root, AppDataType, AppInfo};
+
+    let testenv = TestEnv::new();
+    testenv
+        .command()
+        .args(&["--show-paths"])
+        .assert()
+        .success()
+        .stdout(contains(format!(
+            "Custom pages dir: {}",
+            get_app_root(
+                AppDataType::UserData,
+                &AppInfo {
+                    name: "tealdeer",
+                    author: "tealdeer"
+                }
+            )
+            .map(|path| path.join("pages"))
+            .unwrap()
+            .to_str()
+            .unwrap()
```

Comment:
Just replicates how the Default is defined for custom_pages_dir in `RawDirectoriesConfig`

Reply:
Two notes on this:

1. If it is duplicated, maybe it should be a function?
2. Why is there a `unwrap` after a `map`? The `unwrap` could be moved up and then there wouldn't have to be a closure. You could also make it an `expect` with a message that just says that `get_app_root` is broken or so (which shouldn't happen, but would be nicer)

Reply:
This is just the test which is why

Reply:
Oh, I didn't mean to hardcode it, I meant using `expect` instead of `unwrap` to provide a more useful panic message:

```rust
get_app_root(
    AppDataType::UserData,
    &AppInfo {
        name: "tealdeer",
        author: "tealdeer"
    }
)
.expect("get_app_root failed, this should never happen...")
.join("pages")
.to_str()
.expect("path returned from get_app_root was not valid UTF-8, this shouldd never happen...")
```

About having it in a function: I that there are details (like the arguments of `get_app_root`) which don't really belong into this test case (and would have to be replaced if these would ever change). It's not a must though

Reply:
Oh i see, yeah, I kinda did the map then the unwraps.. which is kind of a strange order, and wasn't quite clear what I was unwrapping. Changed it to expect() calls like how you showed.